odxtools/diaglayer.py

```diff
@@ -313,11 +311,6 @@ def global_negative_responses(self) -> NamedItemList[Response]:
         """All global negative responses applicable to this DiagLayer"""
         return self._global_negative_responses
 
-    @property
-    def tables(self) -> NamedItemList[Table]:
```

Comment:
The `diaglayer.tables` property is gone, how is the user supposed to access tables now?

Reply:
via `diag_layer.diag_data_dictionary_spec.tables`. You have a point though, I should add a deprecated method which does that. (fixed: https://github.com/mercedes-benz/odxtools/pull/164/commits/bdc56ee9dc763de6e6735f138c97dfa188d1dafe )